src/server.py

```diff
@@ -377,13 +352,12 @@ def run_jobtypes_query():
 def run_seta_query():
     start_date, end_date = clean_date_params(request.args, delta=SETA_WINDOW)
 
-    db = create_db_connnection()
-    cursor = db.cursor()
-    query = "select bugid, platform, buildtype, testtype, duration from testjobs \
-             where failure_classification=2 and date>='%s' and date<='%s'" % (start_date, end_date)
-    cursor.execute(query)
+    data = session.query(Testjobs.bugid, Testjobs.platform, Testjobs.buildtype, Testjobs.testtype,
+                         Testjobs.duration).filter(and_(Testjobs.failure_classification == 2,
+                                                        Testjobs.date >= start_date,
+                                                        Testjobs.date <= end_date)).all()
```

Comment:
we should enlarge the date range- it is most important on the end date to get the latest failures.